### ISSUE TITLE
CSP errors related to typekit

### ISSUE THREAD

**Issue by jfrankl** [PublicMapping/publicmapping.github.com#10]
Getting some CSP errors when I open the site:

```
(index):1 Refused to load the stylesheet 'https://p.typekit.net/p.css?s=1&k=xee7ugv&ht=tk&f=13454.13455.13458.13459.30194.30195.30196.30198.30201.30203&a=9869364&app=typekit&e=css' because it violates the following Content Security Policy directive: "style-src 'self' 'unsafe-inline' use.typekit.net". Note that 'style-src-elem' was not explicitly set, so 'style-src' is used as a fallback.

(index):1 Refused to load the font 'https://use.typekit.net/af/1709eb/000000000000000000010b60/27/l?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=n7&v=3' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'font-src' was not explicitly set, so 'default-src' is used as a fallback.

(index):1 Refused to load the font 'https://use.typekit.net/af/1709eb/000000000000000000010b60/27/d?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=n7&v=3' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'font-src' was not explicitly set, so 'default-src' is used as a fallback.
```

According to [this post](https://security.stackexchange.com/a/212921), Adobe recommends the following:

```
script-src 'self' use.typekit.net;
style-src 'self' 'unsafe-inline' use.typekit.net;
img-src 'self' p.typekit.net;
connect-src performance.typekit.net
```

Site should look like this when fonts are loaded:
![image](https://user-images.githubusercontent.com/1809908/88724175-115d1080-d0f8-11ea-8a5a-321e4e1b09fa.png)

**Comment by hectcastro** [comment]
I think resolved by 75780c40f3985a1436e3e86d1f186f294e1ad814 and 36593cf9b4b907b13b76320af4f4fba7ca2acda7.

**Comment by jfrankl** [comment]
Oh yeah, it seems to work now! Thank you.
